### PR TITLE
WildFly 19.0.0 for MP 3.2

### LINKED ISSUE #233: Add Wildfly as a MicroProfile runtime for Starter
Please add Wildfly to the list of "MicroProfile servers". Wildfly is "in progress" under the MicroProfile implementations page:
https://wiki.eclipse.org/MicroProfile/Implementation

Thank you,
Stefano

### PR DESCRIPTION
**Pull request by Karm**
Fixes #233 
Adds [WildFly 19](https://wildfly.org/news/2020/03/18/WildFly19-Final-Released/)

### CODE REVIEW

File: src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/server/WildFlyServer.java [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        Xpp3Dom layer = new Xpp3Dom("layer");
+        layer.setValue("jaxrs");
+        layers.addChild(layer);
+
+        if (microprofileSpecs.contains(MicroprofileSpec.CONFIG)) {
+            layer = new Xpp3Dom("layer");
+            layer.setValue("microprofile-config");
+            layers.addChild(layer);
+        }
+        boolean mpConfigAdded = false;
```
Comment (rdebusscher) [inline src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/server/WildFlyServer.java]: not used (only set in line 143) and in comment on line 169

**Comment by Karm** [comment]
@rdebusscher Thank you for the review. I deleted the variable and also some obsoleted comments.

**Comment by jfdenise** [comment]
@Karm , I have just released 1.0.0.Alpha4 of the plugin, would be great if you could upgrade. I ran mvn clean verify -Pthorntail -Dtest=TestMatrixTest#wildfly* -Dskip.integration.tests=false -DSTARTER_TS_WORKSPACE=/tmp and all went well.

**Comment by Karm** [comment]
@jfdenise Upgraded in the last commit.

File: src/it/java/org/eclipse/microprofile/starter/utils/MPSpec.java [hunk 2]
```diff
@@ -50,7 +50,7 @@
     }),
     OPEN_API(new String[][]{
             // Tomee shows only /resilience:, all others show "/data/resilience".
-            new String[]{"openapi", "/resilience"}
+            new String[]{"openapi", "/hello"}
```
Comment (Emily-Jiang) [inline src/it/java/org/eclipse/microprofile/starter/utils/MPSpec.java]: Why did you change this url?

Comment (Karm) [reply src/it/java/org/eclipse/microprofile/starter/utils/MPSpec.java]: @Emily-Jiang Because /resilience is not present if just OpenAPI is selected, although /hello default endpoint is always present.
